src/framework/mlt.vers

```diff
@@ -663,4 +663,8 @@ MLT_7.14.0 {
   global:
     mlt_producer_probe;
     mlt_chain_attach_normalizers;
+    mlt_deinterlacer_name;
+    mlt_deinterlacer_id;
+    mlt_link_filter_init;
+    mlt_link_filter_metadata;
```

Comment:
Please change these to be in a new block `MLT_7.16.0` derived from `MLT_7.14.0`.

Reply:
done